lib/waterline/error/WLError.js

```diff
@@ -16,128 +16,15 @@ var _ = require('lodash');
  * @param  {Object} properties
  * @constructor {WLError}
  */
-function WLError( properties ) {
-
-  // Call super constructor (Error)
-  WLError.super_.call(this);
-
-  // Fold defined properties into the new WLError instance.
-  properties = properties||{};
-  _.extend(this, properties);
-
-  // Generate stack trace
-  // (or use `originalError` if it is a true error instance)
-  if (_.isObject(this.originalError) && this.originalError instanceof Error) {
-    this._e = this.originalError;
-  }
-  else this._e = new Error();
-
-  // Doctor up a modified version of the stack trace called `rawStack`:
-  this.rawStack = (this._e.stack.replace(/^Error(\r|\n)*(\r|\n)*/, ''));
```

Comment:
What's the advantage of doing this in the first place, and what are the consequences of removing this? Have we codified knowledge of rawstack anywhere in our codebase or patterns?